### DISCUSSION TITLE
Lambda Artillery Couldnt Burst beyond 3000 TPS

### DISCUSSION THREAD

**Discussion by VincentV89** [artilleryio/artillery#1624] [Artillery]
Hi,

I've been using lambda artillery feature with great ease. My issue is that I could only reach a maximum of 3000 TPS even if my arrival rate is higher (i.e 10,000).

How do I scale to arrivalRate/TPS beyond 200k RPS/400k RPS vertically or horizontally as mentioned in the [documentation](https://www.artillery.io/blog/open-source-distributed-load-testing-with-lambda#:~:text=Each%20AWS%20Lambda%20is%20limited,15m%20bursts%20of%20high%20traffic.)?

Regards,
Vincent

**Comment by hassy** [comment]
hi @VincentV89 👋 

> I've been using lambda artillery feature with great ease. 

Woo hoo! Love hearing that! 😄 

> How do I scale to arrivalRate/TPS beyond 200k RPS/400k RPS vertically or horizontally

The best way to do it is to reduce the `arrivalRate` in the script and increase the number of Lambdas with the `--count` flag, e.g.:

```
artillery run --platform aws:lambda --count 100 myscript.yml
```
